numba_dppy/tests/_helper.py

```diff
@@ -123,6 +123,34 @@ def is_windows():
     pytest.param("level_zero:gpu:0", marks=skip_no_level_zero_gpu),
 ]
 
+skip_opencl_gpu_not_default_device = pytest.mark.skipif(
+    "opencl:gpu" not in dpctl.get_current_queue().sycl_device.filter_string,
```

Comment:
If `has_opencl_gpu()` is already returning `False`, then performing this check has no point, it's going to be `False` anyway. Not clear what is going on here.